src/EnergyPlus/HVACManager.cc

```diff
@@ -72,6 +72,7 @@
 //#include <CoolTower.hh>
 #include <DataAirflowNetwork.hh>
 #include <DataAirLoop.hh>
+#include <DataBranchAirLoopPlant.hh>
 #include <DataContaminantBalance.hh>
 #include <DataConvergParams.hh>
 #include <DataEnvironment.hh>
```

Comment:
I'm confused, here's what I see in this branch. DataBranchAirLoopPlant.hh isn't there ?? Something's wrong.

    //#include <CoolTower.hh>
    #include <DataAirflowNetwork.hh>
    #include <DataAirLoop.hh>
    #include <DataContaminantBalance.hh>
    #include <DataConvergParams.hh>
    #include <DataEnvironment.hh>